File: Code Examples/JWTSecurityMongo/src/main/java/io/hashimati/security/domains/LoginEvent.java
package io.hashimati.security.domains;

import io.micronaut.core.annotation.Introspected;
import lombok.Data;

import java.util.Date;

@Data
@Introspected
public class LoginEvent {
    private String username, password;
    private LoginStatus status;
    private Date lastTryDate;
    private Date lastTimeLogin;
}

File: Code Examples/JWT/Kotlin/JWTKotlinMongo/src/main/kotlin/io/hashimati/security/domains/LoginEvent.kt
package io.hashimati.security.domains

import io.micronaut.core.annotation.Introspected
import java.time.Instant


@Introspected
data class LoginEvent (
 var username: String? = null,
 var password: String? = null,
 var status: LoginStatus? = null,
 var lastTryDate: Instant? = null,
 var lastTimeLogin: Instant? = null,
){

}